src/Microsoft.PowerShell.Commands.Utility/commands/utility/Resolve-ErrorRecord.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Verify approved aliases list.Error occurred in Describe block
```powershell
The term '-or' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At /Users/vsts/agent/2.158.0/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1:152 char:100
```